src/wsrep_provider_v26.cpp

```diff
@@ -568,7 +597,7 @@ wsrep::wsrep_provider_v26::wsrep_provider_v26(
     init_args.connected_cb = &connected_cb;
     init_args.view_cb = &view_cb;
     init_args.sst_request_cb = &sst_request_cb;
-    init_args.encrypt_cb = &encrypt_cb;
+    init_args.encrypt_cb = encrypt_cb;
```

Comment:
To my understanding, we should set `init_args.encrypt_cb = 0` here if the encryption service is not provided.

Reply:
Callback will be set if encryption service is provided & enabled.